torch/jit/__init__.py

```diff
@@ -697,6 +704,21 @@ def __getattr__(self, attr):
             raise AttributeError("'CompilationUnit' has no attribute '{}'".format(attr))
         return r
 
+    def _import(self, src, constants):
+        """ test import logic for single function, use only for testing """
+        src = "op_version_set = 0\n{}".format(src)
+        # HACK: export always adds a self argument
```

Comment:
we're planning to remove this before committing to the serialization format for the release, right?

Reply:
Yeah, I need to do an update to the python printer so that it uses the first-class function as its source for serialization. Once I do that then this extra self argument business disappears.